src/main/java/org/mastodon/views/trackscheme/display/TrackSchemePanel.java

```diff
@@ -177,7 +178,7 @@ public class TrackSchemePanel extends JPanel implements
 	 */
 	private boolean ignoreScrollBarChanges;
 
-	private final TimepointModel timepoint;
+	private final TimepointModel navigationTimepointModel;
```

Comment:
Why do you call this timepointModel navigationTimepointModel? Please undo the renaming if there's no reasoning behind it.

Reply:
There was a reasoning behind it during development. But that reason now does not exist anymore. Thus I undid the renaming.